### PR TITLE
samples: openthread: update harness THCI file

### PR DESCRIPTION
**Pull request by edmont**
Update file used for Thread Certification, which handles OpenThread CLI changes.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

KRKNWK-8614

### CODE REVIEW

**Review by MarekPorwisz — APPROVED** [review]

**Review by lmaciejonczyk — APPROVED** [review]

File: samples/openthread/cli/harness/NCS.py [hunk 1]
```diff
@@ -16,14 +16,7 @@
...
 #    names of its contributors may be used to endorse or promote products
-#    derived from this software without specific prior written permission.
-#
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
-# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#    derived from this software witnRF52840 BUT NOT LIMITED TO, PROCUREMENT OF
```
Comment (pszkotak) [inline samples/openthread/cli/harness/NCS.py:19]: something wrong happened here

Comment (edmont) [reply samples/openthread/cli/harness/NCS.py]: Thanks, fixed.

**Comment by edmont** [comment]
`checkpath` warnings (`TYPO_SPELLING`) from the compliance checks can't be removed since those are method implementations of a third party class.

**Review by pszkotak — APPROVED** [review]

**Review by ru-fu — APPROVED** [review]
We try not to use the NCS abbreviation, so if we can rename the files to nRF_Connect_SDK.* that would be preferable - but I don't know if there are conventions for the file names.

**Comment by edmont** [comment]
> We try not to use the NCS abbreviation, so if we can rename the files to nRF_Connect_SDK.* that would be preferable - but I don't know if there are conventions for the file names.

I'll verify if that's possible and modify the PR in that case.

**Comment by ru-fu** [comment]
Great, thank you! :)

**Comment by edmont** [comment]
Running compliance check passes locally:
`$ZEPHYR_BASE/scripts/ci/check_compliance.py -c d37d414413f79cce1a9b113e09c9a5d99f82f05a -m checkpatch`

How to avoid the typo warnings?

**Pull request merged**